docs/wiki/class-names.md

```diff
@@ -20,8 +20,24 @@ title: Class Names
 - `ACRE_VRC111` - AN/VRC-111
 - `ACRE_SEM90` - SEM90
 
-### Ground Spike Antennas
+### Ground Spikes
 
 - `ACRE_VHF30108SPIKE` - VHF30108 GS
 - `ACRE_VHF30108` - VHF30108 GSM
 - `ACRE_VHF30108MAST` - VHF30108 Mast
+
+### Antennas
+
+- `ACRE_100CM_VHF_TNC` - 1M VHF TNC
+- `ACRE_100CM_VHF_BNC` - 1M VHF BNC
+- `ACRE_2HALFINCH_UHF_TNC` - 2.5 INCH UHF PRC343
+- `ACRE_SEM52_SHORT_BNC` - 0.4M SEM52/SEM70
+- `ACRE_SEM52_LONG_BNC` - 0.9M SEM52
+- `ACRE_SEM70_LONG_BNC` - 1.03M SEM70
+- `ACRE_AT271_38` - AT-271 PRC-77
+- `ACRE_123CM_VHF_TNC` - 1.23M VHF TNC
+- `ACRE_270CM_VEH_BNC` - 2.7M MB VEH BNC
+- `ACRE_FA80_BNC` - FA80 VEH BNC
+- `ACRE_AS1729_38` - AS-1729/VRC
+- `ACRE_243CM_VHF_TNC` - 2.43M Ground Spike
```

Comment:
This gets confusing with the tittle separation. It is an antenna but its description is ground spike

Reply:
These are all using `shortName` from the antenna config:
https://github.com/IDI-Systems/acre2/blob/29e20ea74051eb1274735c8d28a04fa55eaf2ba7/addons/sys_antenna/CfgAcreComponents.hpp#L121